shell/browser/electron_browser_main_parts_linux.cc

```diff
@@ -1,23 +0,0 @@
-// Copyright (c) 2022 GitHub, Inc.
-// Use of this source code is governed by the MIT license that can be
-// found in the LICENSE file.
-
-#include "shell/browser/electron_browser_main_parts.h"
-
-#include "base/command_line.h"
-#include "base/environment.h"
-
-namespace electron {
-
-void ElectronBrowserMainParts::SetDesktopStartupId() {
-  // TODO(clavin): this was removed upstream in
-  // https://chromium-review.googlesource.com/c/chromium/src/+/6819616 but it's
-  // not clear if that was intentional.
-  auto const env = base::Environment::Create();
-  auto* const command_line = base::CommandLine::ForCurrentProcess();
-  if (std::optional<std::string> desktop_startup_id =
-          env->GetVar("DESKTOP_STARTUP_ID"))
-    command_line->AppendSwitchASCII("desktop-startup-id", *desktop_startup_id);
```

Comment:
Code paths that rely on the flag value added via https://chromium-review.googlesource.com/c/chromium/src/+/3651517 still exist in upstream, we haven't consumed this flag in our startup path so this would be a noop but can you follow-up with the author what would be the significance of the `PlatformWindow::NotifyStartupComplete` code paths in upstream now that the flag is removed ?